File: app/src/main/java/dev/ragnarok/fenrir/model/Career.java
package dev.ragnarok.fenrir.model;

public class Career {

    private Community group;

    private String company;

    private int countryId;

    private int cityId;

    private int from;

    private int until;

    private String position;

    public int getCityId() {
        return cityId;
    }

    public Career setCityId(int cityId) {
        this.cityId = cityId;
        return this;
    }

    public int getCountryId() {
        return countryId;
    }

    public Career setCountryId(int countryId) {
        this.countryId = countryId;
        return this;
    }

    public int getFrom() {
        return from;
    }

    public Career setFrom(int from) {
        this.from = from;
        return this;
    }

    public Community getGroup() {
        return group;
    }

    public Career setGroup(Community group) {
        this.group = group;
        return this;
    }

    public int getUntil() {
        return until;
    }

    public Career setUntil(int until) {
        this.until = until;
        return this;
    }

    public String getCompany() {
        return company;
    }

    public Career setCompany(String company) {
        this.company = company;
        return this;
    }

    public String getPosition() {
        return position;
    }

    public Career setPosition(String position) {
        this.position = position;
        return this;
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/model/Career.kt
package dev.ragnarok.fenrir.model

class Career {
    var group: Community? = null
        private set
    var company: String? = null
        private set
    var countryId = 0
        private set
    var cityId = 0
        private set
    var from = 0
        private set
    var until = 0
        private set
    var position: String? = null
        private set

    fun setCityId(cityId: Int): Career {
        this.cityId = cityId
        return this
    }

    fun setCountryId(countryId: Int): Career {
        this.countryId = countryId
        return this
    }

    fun setFrom(from: Int): Career {
        this.from = from
        return this
    }

    fun setGroup(group: Community?): Career {
        this.group = group
        return this
    }

    fun setUntil(until: Int): Career {
        this.until = until
        return this
    }

    fun setCompany(company: String?): Career {
        this.company = company
        return this
    }

    fun setPosition(position: String?): Career {
        this.position = position
        return this
    }
}